recipes/sigslot/all/test_package/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.1)
 project(test_package)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(TARGETS)
```

Comment:
Why TARGETS?

Reply:
To avoid `include_directories`.
This is done via: `conan_basic_setup` -> `conan_global_flags`.

Reply:
But `conan_basic_setup()` is enough, no need to call `conan_basic_setup(TARGETS)`

Reply:
Yes, but I want to test the targets, not conan's global include_directories.
It's the same reason I have changed `${CONAN_LIBS}` with `Pal::Sigslot`.

Reply:
`Pal::Sigslot` comes from module file generated by `cmake_find_package`, not from conan_basic_setup.

Reply:
Try this yourself with sigslot:
- run `conan_basic_setup()`
- don't do any `target_link_libraries(test_package ...)`

And report back.

Reply:
Ok I understand know. Didn't know that `conan_basic_setup` was doing a global include of headers directories.

Since `sigslot` is header only library, `target_link_libraries` is useless and doesn't test anything if using `conan_basic_setup()`. Thanks.